web-components/src/components/modal/Modal.ts

```diff
@@ -271,8 +272,8 @@ export namespace Modal {
                   id="${this.htmlId}"
                   class="md-modal ${classMap(this.modalContainerClassMap)}"
                   aria-label="${this.ariaLabel}"
-                  aria-labelledby="${this.ariaLabelledBy}"
-                  aria-describedby="${this.ariaDescription}"
+                  aria-labelledby="modal_header"
+                  aria-describedby=${ifDefined(this.ariaDescription || undefined)}
```

Comment:
Why to add undefined at the end? Check the proper usage of ifDefined

Reply:
updated